From bb7800494d834f517c0d97768ef3cc1d2521d4eb Mon Sep 17 00:00:00 2001
From: Andrea Adami <andrea.adami@gmail.com>
Date: Tue, 15 Jun 2021 01:41:13 +0200
Subject: [PATCH 1/1] streamproxy: fix build with musl

fix
 | ../../git/src/webrequest.cpp:15:10: fatal error: sys/unistd.h:
 No such file or directory
 |    15 | #include <sys/unistd.h>

Signed-off-by: Andrea Adami <andrea.adami@gmail.com>
---
 src/webrequest.cpp | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/webrequest.cpp b/src/webrequest.cpp
index ce20a45..02e0f24 100644
--- a/src/webrequest.cpp
+++ b/src/webrequest.cpp
@@ -12,7 +12,11 @@ using std::string;
 #include <string.h>
 #include <sys/types.h>
 #include <sys/stat.h>
+#ifdef __GLIBC__
 #include <sys/unistd.h>
+#else
+#include <unistd.h>
+#endif
 #include <string.h>
 
 WebRequest::WebRequest(const ConfigMap &config_map_in, const HeaderMap &headers_in,
-- 
2.17.1

